packages/typespec-ts/test/modularIntegration/generated/lro_modular/generated/src/common/lroImpl.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Not sure if common folder is a good place to put this, but I am trying to remove common folder interfaces.ts

Reply:
Okay, a more common question is where to put the internal shared stuff, do you have any suggestion? 

If no I may move them to `api` folder. I put them to common folder because I'd like to all apis could share this LRO helper function.